rotkehlchen/tests/conftest.py

```diff
@@ -235,7 +235,7 @@ def vcr_cassette_dir(request: pytest.FixtureRequest) -> str:
 
         # verify that the cassettes are present locally and up to date
         if (tests_caching_dir / '.git').exists() is False:
-            cmd = f'git clone https://github.com/rotki/test-caching.git {tests_caching_dir}'
+            cmd = f'git clone https://github.com/rotki/test-caching.git "{tests_caching_dir}"'
```

Comment:
Can you move this change in a different commit and make it to bugfixes? We need it in oth places if it fixes something.